test/controllers/CheckYourAppealController.test.ts

```diff
@@ -20,7 +20,7 @@ import { createFakeSession } from 'test/utils/session/FakeSessionFactory';
 
 const config = getDefaultConfig();
 
-const appeal = {
```

Comment:
Same here, may I ask you to move that to separate PR. It feels like distraction :(